src/mmrelay/plugins/base_plugin.py

```
@@ -543,101 +558,118 @@ async def send_matrix_message(self, room_id, message, formatted=True):
+    def _require_plugin_name(self) -> str:
+        """
+        Return the plugin name, enforcing that it has been initialized.
 
-        Args:
-            meshtastic_id (str): Node identifier
-            node_data: Data to store (single item or list)
+        Raises:
+            ValueError: If plugin_name has not been set by initialization.
+        """
+        if self.plugin_name is None:
+            raise ValueError("Plugin name not initialized")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)